### DISCUSSION TITLE
Style TabView

### DISCUSSION THREAD

**Discussion by andyzukunft** [enisn/UraniumUI#722] [Q&A] [answered]
I am trying to use the TabView component and want to style the tab header colors. How would you recommend on doing that?

The documentation (https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/TabView) is replacing the whole button with a custom implementation - which would require to add logic for selection, Commands etc.

Is there a way to only style the header according to its state (selected / not selected?

**Comment by enisn** [comment, marked as answer]
Currently, the default template implemented with [DataTriggers](https://learn.microsoft.com/en-us/dotnet/maui/fundamentals/triggers?view=net-maui-8.0#data-triggers). 
https://github.com/enisn/UraniumUI/blob/181f131285ee92e58df6ce1af81671f5d6f9e1ee/src/UraniumUI.Material/Controls/TabView.cs#L38


But it's on my roadmap, I'll make add styling and visual states like treeview: 
https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/TreeView#styles

Unfortunately, there is no styling logic for now

**Reply by andyzukunft** [reply]
Thanks for your reply. I will wait for a more simple styling behaviour and keep my layout as it is for the moment.

**Comment by sidibeh2563** [comment]
I do have issues when I use CommunityToolkit MVVM with Tabview Button.Triggers. It looks for 'Command' and 'Title' in the viewmodel  (BindingContext) rather than in Materia: For example `<Button Text="{Binding Title}"  Command="{Binding Command}">`. Do you have any idea what is the walk around for this.
